tests/sanic/test_view.py

```diff
@@ -0,0 +1,117 @@
+from random import random
+
+import pytest
+
+import strawberry
+from sanic import Sanic
+from strawberry.sanic.views import GraphQLView as BaseGraphQLView
+from strawberry.types import ExecutionResult, Info
+
+
+def create_app(**kwargs):
+    @strawberry.type
+    class Query:
+        hello: str = "strawberry"
+
+    schema = strawberry.Schema(query=Query)
+
+    class GraphQLView(BaseGraphQLView):
+        def get_root_value(self):
+            return Query()
+
+    app = Sanic("test-app-" + str(random()))
```

Comment:
In the other tests, you hardcode the App name. Should this be made consistent, or did you need  the random name for a reason?

Reply:
In this context, hardcoded app name is required so that we can run multiple iterations of the test in parallel using a single test rig.